app/src/main/java/com/codepath/apps/critterfinder/models/SearchFilter.java

```diff
@@ -46,28 +46,12 @@ public String toString() {
     }
 
     public enum Species {
-        DOG,
-        CAT
-    }
-
-    public enum Age {
-        BABY,
-        YOUNG,
-        ADULT,
-        SENIOR
-    }
-
-    // TODO - this should become a first level model populated by the
-    // breeds end point.
-    // A filter should work with an array of breeds
-    public enum Breed {
-        COTON("Coton Du Tulear"),
-        KINGCHARLES("King Charles Cavalier"),
-        HAVANESE("Havanese");
+        DOG("dog"),
+        CAT("cat");
```

Comment:
turns out pet finder is case sensitive! dog and cat work but DOG and CAT do not.